apps/server/src/modules/learnroom/service/common-cartridge-export.service.ts

```diff
@@ -1,172 +1,93 @@
-import { LessonService } from '@modules/lesson/service';
-import { TaskService } from '@modules/task/service';
+import { LessonService } from '@modules/lesson';
+import { TaskService } from '@modules/task';
 import { Injectable } from '@nestjs/common';
-import { ComponentProperties, Course, Task } from '@shared/domain/entity';
+import { ComponentProperties } from '@shared/domain/entity';
 import { EntityId } from '@shared/domain/types';
-import { ComponentType } from '@src/shared/domain/entity/lesson.entity';
 import {
 	CommonCartridgeFileBuilder,
-	CommonCartridgeIntendedUseType,
-	CommonCartridgeResourceType,
+	CommonCartridgeOrganizationBuilder,
 	CommonCartridgeVersion,
-	ICommonCartridgeResourceProps,
-	ICommonCartridgeWebContentResourceProps,
-} from '../common-cartridge';
-import { createIdentifier } from '../common-cartridge/utils';
+} from '../../common-cartridge';
+import { CommonCartridgeMapper } from '../mapper/common-cartridge.mapper';
 import { CourseService } from './course.service';
 
 @Injectable()
 export class CommonCartridgeExportService {
-	constructor(
+	public constructor(
```

Comment:
Interessting i think the constructor is always public like the methods. Currently we want always explicit the public to all methods, but do not add this to the constructor. But we do not think about it before. I am unsure if private work and can be a valid use case.

I will check it later, but please remove it for now to hold it consistent to the rest of the code.

Reply:
Done